context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FileTripletSystemMonitor.java

```diff
@@ -415,38 +171,4 @@ protected final void handleFile(final String fileName, final Long time,
         }
         return this.filePair;
```

Comment:
`updateFilePair` can be void because it only modifies `this.filePair` and the return value is never used.

It should probably be renamed to `updateFileTriplet`.

Reply:
[x] done